Ed-Fi-Exchange-OSS/EdFi-Project-Buzz
# [FIF-347] Add IIS support for UI

FIF-347 adds IIS support to the install process. It also updates the README in the installer to add a requirement for IIS configuration. Added web.config for React routes.

## TEST

- Build latest NuGet packages using test.ps1
- Create a IIS website for Buzz. Use 4202 for the port and http. Use C:\Ed-Fi\Buzz\UI\build as the file location. 
- Update the configuration.json values. We will assume Google SSO and whatever your configuration is that works currently on your local:
  - includePrerelease = true (to get prereleases)
  - artifactRepo = C:\nugetRepo (to get your local built packages)
  - installPath = C:\Ed-Fi\Buzz (as in the IIS configuration
  - ui.url = http://localhost:4202 (from the IIS config)
  - ui.port = 4202
- Run the install.ps1 from the installer\eng\win folder.
- Expect that your install should report services running...
```bash
Checking Ed-Fi Buzz App Services and Webs statuses ...

Name                           Value
----                           -----
Ed-Fi Buzz UI Website          Status code returned: 200
Ed-Fi Buzz API Service         Running
Ed-Fi Buzz ETL Service         Running
Ed-Fi Buzz API Website         Status code returned: 200
```
- Test http://localhost:4202 - you should see the Owl logo, and should be able to authenticate and see the roster.
